docs/go/workflows.md

```diff
@@ -247,7 +247,23 @@ The `workflow.ExecuteChildWorkflow` call is used to schedule Workflow Executions
 <!--SNIPEND-->
 
 By default, a Child Workflow Execution inherits the options provided to the Parent Workflow Execution, and the Temporal Server will automatically generate a Child Workflow ID.
-You can overwrite these options and specify a customer Child Workflow ID by customizing `ChildWorkflowOptions` and adding them to the execution context.
+You can overwrite any of these options and specify a customer Child Workflow ID by customizing `ChildWorkflowOptions` and adding them to the execution context.
```

Comment:
Should be "custom" instead of "customer"?

```suggestion
You can overwrite any of these options and specify a custom Child Workflow ID by customizing `ChildWorkflowOptions` and adding them to the execution context.
```